packages/core/src/SlideBox/styles.ts

```diff
@@ -0,0 +1,16 @@
+/* istanbul ignore file */
+
+import styled from 'styled-components'
+import { Flex, Box } from '..'
+
+const SlideWrapper = styled(Box)``
+
+const ScrollFlex = styled(Flex)`
+  position: relative;
+  overflow: auto;
+  > ${SlideWrapper} {
```

Comment:
should this have an `&`?

Reply:
Just tested - seems to work both ways. I'll change to `&>`